internal/fcs/cloud_aws_account.go

```diff
@@ -1552,6 +1753,31 @@ func (r *cloudAWSAccountResource) ValidateConfig(
 		return
 	}
 
+	// Validate S3 log ingestion requirements
+	if config.RealtimeVisibility != nil && !config.RealtimeVisibility.LogIngestionMethod.IsNull() {
```

Comment:
`.IsNull` will return false if the value is unknown which means the if statement could pass when `LogIngestionMethod` is unknown.
```suggestion
	if config.RealtimeVisibility != nil && utils.IsKnown(config.RealtimeVisibility.LogIngestionMethod) {
```

`utils.IsKnown` will return true if a value IS NOT unknown and IS NOT null.